### PR TITLE
Use "Back to top" component in article templates

### PR DESCRIPTION
**Pull request by bkjohnson**
## Description

Part of https://github.com/department-of-veterans-affairs/va.gov-team/issues/31200 and a continuation of #759 

This PR uses `<va-back-to-top>` in 3 more templates:

- `src/site/layouts/page.drupal.liquid`
- `src/site/layouts/vet_center.drupal.liquid`
- `src/site/layouts/vet_center_locations_list.drupal.liquid`

These templates have an `<article>` as the main content container, and there are [styles in `vets-website`](https://github.com/department-of-veterans-affairs/vets-website/blob/1e5405d32acc6caced2fae6f5c20fd175ccaf06d/src/platform/site-wide/sass/shame.scss#L64-L68) which add padding that interferes with the overall design of `<va-back-to-top>`. 

This PR also adds a design system utility class to the `<article>` in order to overcome this additional padding. The number of `<article>`s in `vets-website` and `content-build` is fairly high, so even though it would be preferable to remove the CSS from the shame file which causes the extra padding, doing so without investigating every occurrence of `<article>` would be risky without the nontrivial work required to ensure that the removed CSS doesn't cause any regressions.

```
✔︎ content-build (btt-article-templates) ✗ rg "<article" -l | count
41
```
```
✔︎ vets-website (master) ✗ rg "<article" -l | count
20
```

## Testing done

Local testing :eyes: 

## Screenshots

Attached to each template

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/site/layouts/page.drupal.liquid [hunk 1]
```diff
@@ -66,16 +66,16 @@
               </div>
             </div>
           {%  endif %}
+          {% if lang != "en" %}
+            <va-back-to-top></va-back-to-top>
```
Comment (bkjohnson) [inline src/site/layouts/page.drupal.liquid:70]: Sample page:

`disability/eligibility/hazardous-materials-exposure/agent-orange/thailand-military-bases/index.html`

## Before (staging)

image

## After

image

File: src/site/layouts/vet_center.drupal.liquid [hunk 2]
```diff
@@ -158,12 +158,11 @@
               %}
             </div>
           {% endif %}
+          <va-back-to-top></va-back-to-top>
```
Comment (bkjohnson) [inline src/site/layouts/vet_center.drupal.liquid:161]: Sample page:

`yuma-vet-center/index.html`

## Before (staging)

image

## After

image

File: src/site/layouts/vet_center_locations_list.drupal.liquid [hunk 3]
```diff
@@ -65,13 +65,11 @@
               http://www.va.gov/find-locations.
             </a>
           </p>
+          <va-back-to-top></va-back-to-top>
```
Comment (bkjohnson) [inline src/site/layouts/vet_center_locations_list.drupal.liquid:68]: Sample page:

`yuma-vet-center/locations/index.html`

## Before (staging)

image

## After

image

**Review by bkjohnson — COMMENTED** [review]
The `vads-u-padding-bottom--0` [utility class](https://design.va.gov/utilities/padding#padding-bottom) is added to each `<article>` to remove the extra padding coming from `vets-website`.

**Review by kelsonic — APPROVED** [review]
Thanks so much for doing the before/after screenshots! Makes it super easy to review 💪 Nice work!

**Pull request merged**